app/controllers/application_controller/feature_flags_dependency.rb

```diff
@@ -37,4 +37,9 @@ def download_repositories_enabled?
     logged_in? && current_user.feature_enabled?(:download_repositories)
   end
   helper_method :download_repositories_enabled?
+
+  def repository_import_webhook_enabled?
+    logged_in? && current_user.feature_enabled?(:repository_import_webhook)
+  end
```

Comment:
I've added a feature flag so we can roll this out to some % of actors to try it out.